src/index.html

```diff
@@ -1,14 +1,51 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Enclosures</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Enclosures</h1>
-</body>
+
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Enclosures</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+
+  <body>
+    <div class="family grandfather" id="qa-block">
+      <div class="family father">
+        <div class="family son">
+          <p class="text">
+            In programming languages, a closure is a technique for implementing lexically scoped name binding in a
+            language with first-class functions. Operationally, a closure is a record storing a function together with
+            an environment.
+          </p>
+        </div>
+      </div>
+    </div>
+
+    <div class="family grandfather">
+      <div class="family father">
+        <div class="family son">
+          <p class="text">
+            У програмуванні замиканням називають підпрограму, що виконується в середовищі, яке містить одну або більше
+            зв’язаних змінних. Підпрограма має доступ до цих змінних під час виконання.
+          </p>
+        </div>
+      </div>
+    </div>
+
+    <div class="family grandfather">
+      <div class="family father">
+        <div class="family son">
+          <p class="text">
+            Замыкание в программировании — функция первого класса, в теле которой присутствуют ссылки на переменные,
+            объявленные вне тела этой функции в окружающем коде и не являющиеся её параметрами.
+          </p>
+        </div>
+      </div>
+    </div>
+
+
+  </body>
+
```

Comment:
remove unnecessary blank lines,
an empty line is useful between sibling blocks, not between parent and child blocks